### ISSUE TITLE
Open-Shell Menu : Search box permanent?

### ISSUE THREAD

**Issue by Clemens-Ratte-Polle** [Open-Shell/Open-Shell-Menu#331]
Hi.
Open-Shell Menu 4.4.142

Is it possible to got the search results in the search box to stay a while when working with the results?
I want to edit found images etc. but i can only fetch one of the results and not all of them.
Perhaps an option to make the start menu OR an extra top floating search box durable by checkbox would be useful :)
thx

**Comment by antonyupward** [comment]
I agree.  The ability to have the search results stay worked in XP, but on Win10 the results disappear as soon as one click's away from the search results.  Significantly less useful now.
